project/MimaExcludes.scala

```diff
@@ -51,6 +51,8 @@ object MimaExcludes {
     ProblemFilters.exclude[ReversedMissingMethodProblem]("org.apache.spark.sql.catalog.Catalog.listTables"),
     // [SPARK-43992][SQL][PYTHON][CONNECT] Add optional pattern for Catalog.listFunctions
     ProblemFilters.exclude[ReversedMissingMethodProblem]("org.apache.spark.sql.catalog.Catalog.listFunctions"),
+    // [SPARK-43999][SQL] Support force finish useless stage when AQE on
```

Comment:
really need this item?

Reply:
I consider remove it now, doesn't look elegant.